### DISCUSSION TITLE
request.node.rep_teardown does not exists

### DISCUSSION THREAD

**Discussion by mikraj01** [pytest-dev/pytest#11112] [Q&A]
I want to know if test  failed at teardown and pass that information to next fixture in teardown process. I've been looking into this example, but problem is that `rep_teardown` never exists. https://docs.pytest.org/en/7.1.x/example/simple.html#making-test-result-information-available-in-fixtures

I can find rep_setup and rep_call but no rep_teardown
![image](https://github.com/pytest-dev/pytest/assets/19708132/1b3c7928-fdf8-4be5-8dc6-8420977379c6)

test.py
```python
import pytest

@pytest.fixture(scope="function")
def two():
    yield
    raise AssertionError

@pytest.fixture(scope="function")
def one(request):
    yield
    # here I want to know if teardown has failed
    assert hasattr(request.node, "rep_teardown")

def test_fixtures(one, two):
    pass
```
conftest.py
```python
import pytest

@pytest.hookimpl(tryfirst=True, hookwrapper=True)
def pytest_runtest_makereport(item, call):
    # execute all other hooks to obtain the report object
    outcome = yield
    rep = outcome.get_result()
    # set a report attribute for each phase of a call, which can
    # be "setup", "call", "teardown"
    setattr(item, "rep_" + rep.when, rep)
```

**Comment by The-Compiler** [comment]
I'd say this is expected - after a `yield` in a fixture *is* the teardown phase, so you're trying to check the status of teardown while the teardown is still ongoing.

**Reply by mikraj01** [reply]
@The-Compiler thanks for the reply. Is there any way to know if teardown of previous fixture(s) has failed ?

**Comment by mikraj01** [comment]
I tried another unsuccessful approach. Teardown of fixture one gets called before hook call.
test.py
```python
import pytest

@pytest.fixture(scope="function")
def two():
    yield
    raise AssertionError

@pytest.fixture(scope="function")
def one():
    yield
    pass

def test_fixtures(one, two):
    pass
```

conftest.py
```python
import pytest

@pytest.hookimpl()
def pytest_exception_interact(node, call, report):
    if call.when == 'teardown' and report.failed:
        # do something here when an exception is raised at fixture teardown
        pass
```
